tty/tiny_tty.c

```diff
@@ -193,7 +194,11 @@ static int tiny_write(struct tty_struct *tty,
 	return retval;
 }
 
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 14, 0)) 
 static int tiny_write_room(struct tty_struct *tty)
+#else
+unsigned int tiny_write_room(struct tty_struct *tty)
```

Comment:
Thanks @QuantumForge !  Should this one remain `static` ?

Reply:
Yes. sorry I missed that. I pushed a correction to fix it.